src/services/TwitterService.ts

```diff
@@ -31,12 +31,14 @@ class TwitterService {
 		}).on("data", listener => this.handleTwitterStream(listener, tweetChannel));
 	}
 
-	handleTwitterStream = async ({ id_str: id, text, extended_tweet }: TwitterStreamListener, tweetChannel: TextChannel): Promise<void> => {
+	handleTwitterStream = async ({ id_str: id, text, user, retweeted_status, extended_tweet }: TwitterStreamListener, tweetChannel: TextChannel): Promise<void> => {
 		const embed = new MessageEmbed();
 		const tweet = extended_tweet?.full_text || text;
+		const tweetId = retweeted_status?.id_str || id;
+		const retweetFromCodeSupport = retweeted_status && user?.id_str === TWITTER_ID ? true : false;
```

Comment:
Do you need `? true : false`? Surely `user?.id_str === TWITTER_ID` results in a `Boolean`?